src/Converter.php

```diff
@@ -179,11 +179,12 @@ public static function convert($opt) {
         $context = self::createContext($opt);
 
         $result = file_get_contents($endpoint, false, $context);
-        if (substr($http_response_header[0], 9, 3) !== '200') { //Check http response code for if the request failed
+         $http_response = substr($http_response_header[0], 9, 3);
+        if ($http_response !== '200') { //Check http response code for if the request failed
             if ($result !== false) { //If a text response was given
                 $decoded = json_decode($result, true);//Decode the json
                 if(array_key_exists('error',$decoded)) {
-                    self::exitWithError($decoded['error']); //Exit with the error provided
+                    self::exitWithError("http code :" .$http_response. "" .$decoded['error'], $http_response ); //Exit with the error provided
```

Comment:
Currently, the middle quotes have no space, so the error message will be combined with the http response code (e.g. "http code :403Invalid Token". I suggest altering to make the output look more readable:
```suggestion
                    self::exitWithError("http error code " . $http_response . ": " . $decoded['error'], $http_response ); //Exit with the error provided
```